front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -1056,6 +1035,7 @@ function TryModalInBuilder({
   );
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
IMHO let's kill the code rather than add this, we risk forgetting it

We can't merge before the slack integration in dropdown is merged anyways, and once it is then the code should be removed

Reply:
@lasryaric is working on it killing will make his life harder